geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/parallel/ParallelWANPersistenceEnabledGatewaySenderDUnitTest.java

```diff
@@ -1599,6 +1599,197 @@ public void testRemoveGatewayFromPersistentRegionOnRestart() throws Throwable {
     }
   }
 
+
+  /**
+   * Enable persistence for GatewaySender. Pause the sender and do some puts in local region. Stop
+   * GatewaySender.
+   * Then start GatewaySender. Check if the remote site receives all the events.
+   */
+  @Test
+  public void testPRWithGatewaySenderPersistenceEnabled_Restart_Gateway() {
```

Comment:
Ideally all of the test names in this PR should succinctly describe 3 things: subject under test, specific condition under test, and what we expect to happen to the subject in this condition. These names are missing the last part. For example, something like: `persistentWanGateway_restartSender_expectAllEventsReceived` and `persistentWanGateway_restartSenderWithCleanQueues_expectNoEventsReceived` would be clearer about what is expected to happen.